doc/source/whatsnew/v0.24.0.rst

```diff
@@ -1593,6 +1593,7 @@ Plotting
 
 - Bug in :func:`DataFrame.plot.scatter` and :func:`DataFrame.plot.hexbin` caused x-axis label and ticklabels to disappear when colorbar was on in IPython inline backend (:issue:`10611`, :issue:`10678`, and :issue:`20455`)
 - Bug in plotting a Series with datetimes using :func:`matplotlib.axes.Axes.scatter` (:issue:`22039`)
+- Bug in :func:`_get_standard_colors` that returned 10 colors when num_colors < 10 and color_type='default' (:issue:`20585`)
```

Comment:
can you change this to be more of a user facing note. IOW more along the lines of OP. write what a user would want to know about this bug fix, rather than the implementation.